gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1321,26 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+void
+MM_ParallelGlobalGC::clearHeap(MM_EnvironmentBase *env, MM_HeapWalkerObjectFunc walkFunction)
+{
+	HeapSizes counter;
+	counter.freeBytes = 0;
+	counter.objectBytes = 0;
+
+	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
+	U_64 startTime = omrtime_hires_clock();
+
+	_heapWalker->allObjectsDo(env, walkFunction, &counter, MEMORY_TYPE_RAM, false, false, true);
+
+	_extensions->globalGCStats.fixHeapForWalkTime += omrtime_hires_delta(startTime, omrtime_hires_clock(), OMRPORT_TIME_DELTA_IN_MICROSECONDS);
```

Comment:
put a comment that while clearing actually requires 2 walks (first to convert dark matter into holes, and then to clear all holes) we actually report it as one pass with cumulative time

Reply:
The reason for the two heap walks is missing in the above comment: `first to convert dark matter into holes, and then to clear all holes`.